ppr-ui/src/composables/address/BaseAddress.vue

```diff
@@ -42,69 +44,80 @@
-                      filled
-                      class="street-address-additional"
-                      :label="streetAdditionalLabel"
-                      rows="1"
-                      v-model="addressLocal.streetAdditional"
-                      :rules="[...schemaLocal.streetAdditional]"
+          <v-textarea
+            autocomplete="new-password"
+            auto-grow
+            filled
+            class="street-address-additional"
+            :label="streetAdditionalLabel"
+            rows="1"
+            v-model="addressLocal.streetAdditional"
+            :rules="[...schemaLocal.streetAdditional]"
           />
         </div>
         <div class="form__row three-column">
-          <v-text-field filled
-                        class="item address-city"
-                        :label="cityLabel"
-                        v-model="addressLocal.city"
-                        :rules="[...schemaLocal.city]"
+          <v-text-field
+            autocomplete="new-password"
+            filled
+            class="item address-city"
+            :label="cityLabel"
+            v-model="addressLocal.city"
+            :rules="[...schemaLocal.city]"
           />
           <v-autocomplete v-if="useCountryRegions(country)"
-                    filled
-                    class="item address-region"
-                    hide-no-data
-                    item-text="name"
-                    item-value="short"
-                    :items="getCountryRegions(country)"
-                    :label="regionLabel"
-                    :menu-props="{ maxHeight: '14rem' }"
-                    :rules="[...schemaLocal.region]"
-                    v-model="addressLocal.region"
+            autocomplete="new-password"
+            filled
+            class="item address-region"
+            hide-no-data
+            item-text="name"
+            item-value="short"
+            :items="getCountryRegions(country)"
+            :label="regionLabel"
+            :menu-props="{ maxHeight: '14rem' }"
+            :rules="[...schemaLocal.region]"
+            v-model="addressLocal.region"
           />
           <v-text-field v-else
-                        filled
-                        class="item address-region"
-                        :label="regionLabel"
-                        v-model="addressLocal.region"
-                        :rules="[...schemaLocal.region]"
+            autocomplete="new-password"
+            filled
+            class="item address-region"
+            :label="regionLabel"
+            v-model="addressLocal.region"
+            :rules="[...schemaLocal.region]"
           />
-          <v-text-field filled
-                        class="item postal-code"
-                        :label="postalCodeLabel"
-                        v-model="addressLocal.postalCode"
-                        :rules="[...schemaLocal.postalCode]"
+          <v-text-field
+            autocomplete="new-password"
```

Comment:
Why do all of the fields have autocomplete="new-password"?

Reply:
this is the only way to prevent the chrome autocomplete from going overtop our autocomplete list atm